docs/src/refs.bib

```diff
@@ -64,3 +86,26 @@ @article{pritchard2024solving
   issn = {0266-5611, 1361-6420},
   doi = {10.1088/1361-6420/ad5583},
 }
+
+@article{strohmer2009randomized,
+  title = {A {{Randomized Kaczmarz Algorithm}} with {{Exponential Convergence}}},
+  author = {Strohmer, Thomas and Vershynin, Roman},
+  year = {2009},
+  month = apr,
+  journal = {Journal of Fourier Analysis and Applications},
+  volume = {15},
+  number = {2},
+  pages = {262--278},
+  issn = {1069-5869, 1531-5851},
+  doi = {10.1007/s00041-008-9030-4},
+  langid = {english}
+}
+@misc{pilanci2014iterative,
+  title = {Iterative {{Hessian}} Sketch: {{Fast}} and Accurate Solution Approximation for Constrained Least-Squares},
+  shorttitle = {Iterative {{Hessian}} Sketch},
+  author = {Pilanci, Mert and Wainwright, Martin J.},
+  year = {2014},
+  publisher = {arXiv},
+  doi = {10.48550/ARXIV.1411.0347},
+}
```

Comment:
I think this reference is listed twice. I think the JMLR version (linked in the PR description) should be the appropriate reference to use:

```
@article{pilanci2016iterative,
  title={Iterative Hessian sketch: Fast and accurate solution approximation for constrained least-squares},
  author={Pilanci, Mert and Wainwright, Martin J},
  journal={Journal of Machine Learning Research},
  volume={17},
  number={53},
  pages={1--38},
  year={2016}
}
```